source/features/toggle-files-button.tsx

```diff
@@ -33,11 +35,20 @@ function addButton(): void {
 	);
 }
 
-function init(): void {
-	const repoContent = select('.repository-content')!;
+async function init(): Promise<void> {
+	const repoContent = (await elementReady('.repository-content'))!;
 	observeElement(repoContent, addButton);
-	delegate(document, '.rgh-toggle-files', 'click', ({delegateTarget}) => {
-		delegateTarget.setAttribute('aria-expanded', String(!repoContent.classList.toggle('rgh-files-hidden')));
+
+	const cacheKey = 'files-list-toggled-off';
+	if (await cache.get<boolean>(cacheKey)) {
+		repoContent.classList.toggle('rgh-files-hidden', true);
+		select('.rgh-toggle-files')!.setAttribute('aria-expanded', 'false');
+	}
+
+	delegate(document, '.rgh-toggle-files', 'click', async ({delegateTarget}) => {
+		const toggleState = repoContent.classList.toggle('rgh-files-hidden');
+		delegateTarget.setAttribute('aria-expanded', String(!toggleState));
```

Comment:
I think this attribute was not correct/useful. Can you drop it from everywhere in this function?

Reply:
Done!